core/trino-main/src/main/java/io/trino/sql/planner/OptimizerConfig.java

```diff
@@ -756,4 +758,17 @@ public OptimizerConfig setUseExactPartitioning(boolean useExactPartitioning)
         this.useExactPartitioning = useExactPartitioning;
         return this;
     }
+
+    public boolean isFuseSubPlan()
+    {
+        return fuseSubPlan;
+    }
+
+    @Config("optimizer.fuse-sub-plan")
```

Comment:
Isn't this should be more descriptive? As in, right now we only support a particular kind of fusing.

Reply:
IMO it's better to have one toggle for all fusing rules that are incoming.